### ISSUE TITLE
Format Document does not work in VS Code

### ISSUE THREAD

**Issue by ggat** [getgauge/gauge-vscode#464]
### What command(s) did you run when you found the bug?

<kbd>Option</kbd> + <kbd>Shift</kbd> + <kbd>F</kbd> or "Format Document" from editor window context menu

### Versions
Gauge VS Code extension version: v0.0.15

#### Gauge (Output of `gauge -v`)
```
Gauge version: 1.1.4
Plugins
-------
html-report (4.0.10)
json-report (0.3.3)
screenshot (0.0.1)
spectacle (0.1.4)
ts (0.0.5)
```

#### VS Code 1.49.2

![image](https://user-images.githubusercontent.com/14810832/94650956-b074d300-0308-11eb-8d50-a39ce03094c5.png)

settings.json
```
...
"files.associations": {
    "*.spec": "gauge",
    "*.cpt": "gauge",
    "*.gauge": "gauge"
  },
...
```

**Comment by sriv** [comment]
Just to clarify - which file were you attempting to format?

**Comment by ggat** [comment]
A spec file

**Comment by sriv** [comment]
Since you are on macOS, I'll let someone else from @getgauge/core take a look.

I can see the formatting work well with gauge-vscode 0.0.15 on linux:

![format](https://user-images.githubusercontent.com/389148/94686226-277d8c00-0348-11eb-8560-5cbfacf38e9e.gif)

**Comment by ggat** [comment]
OK, thanks! Not sure if this is related, `gauge format` works from command line

**Comment by zabil** [comment]
I checked it on the following and it's working. Is there anything else configured which might be causing the problem?

```
Gauge version: 1.1.4
Plugins
-------
html-report (4.0.12)
java (0.7.10)
js (2.3.12)
screenshot (0.0.1)
ts (0.1.0)

VS Code Version: 1.49.2
Node v14.10.1
Gauge VS Code plugin v0.0.15
Catalina 10.15.6 (19G2021)
```

**Comment by ggat** [comment]
Thanks for fast responses. I've just disabled all other extensions reloaded IDE and I got message similar to "prettier was set as a formatter, but is not available" after click on the message ballon, I was presented with quick pick menu with only option "Gauge" picked  the option and formatting started working. 

Still not sure if this updated file association, as they look good originally too. 

No I'm unable to reproduce the issue.
